### PR TITLE
Improve compliant example for python/pytorch-redundant-softmax@v1.0

### PR DESCRIPTION
**Pull request by ankit-amazon**
Addressed review comments.

### CODE REVIEW

File: src/python/detectors/pytorch_redundant_softmax/pytorch_redundant_softmax.py [hunk 1]
```diff
@@ -96,90 +96,50 @@ def test_loop(dataloader, model, loss_fn):
...
-            for x, y in dataloader:
-                pred = model(x)
-                test_loss += loss_fn(pred, y).item()
-                correct += (pred.argmax(1) == y).type(torch.float).sum().item()
-
-        test_loss /= num_batches
-        correct /= size
-        print(f"Test Error: \n Accuracy: {(100 * correct):>0.1f}%, "
-              f"Avg loss: {test_loss:>8f} \n")
+            loss_fct = nn.CrossEntropyLoss(weight=torch.Tensor(
```
Comment (linghuiluo) [inline src/python/detectors/pytorch_redundant_softmax/pytorch_redundant_softmax.py:137]: need to add a comment about this case is compliant

**Review by grl-wang — APPROVED** [review]
